backend/hitas/views/apartment.py

```diff
@@ -23,18 +23,18 @@
     ApartmentConstructionPriceImprovement,
     ApartmentMaximumPriceCalculation,
     Building,
+    ConstructionPriceIndex,
+    ConstructionPriceIndex2005Equal100,
     HousingCompany,
+    MarketPriceIndex,
+    MarketPriceIndex2005Equal100,
     Owner,
     Ownership,
     SurfaceAreaPriceCeiling,
-    MarketPriceIndex,
-    MarketPriceIndex2005Equal100,
-    ConstructionPriceIndex,
-    ConstructionPriceIndex2005Equal100,
 )
 from hitas.models._base import HitasModelDecimalField
 from hitas.models.apartment import ApartmentMarketPriceImprovement, ApartmentState, DepreciationPercentage
-from hitas.utils import this_month, RoundWithPrecision
+from hitas.utils import RoundWithPrecision, this_month
```

Comment:
And this is why the pre-commit hook is amazing (There would be no separate commit for this) 😄